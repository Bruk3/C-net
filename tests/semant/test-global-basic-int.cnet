

int x = 9 * 5;
int y = 3 - 9;
int z = 4 * 9 - 1 - 12 % 5;


int a = 1 && 0;
int b = 1 || 0;


int c = 15 == 9;
int d = 2 == 2;

int main () {
	
	return 0;
}
