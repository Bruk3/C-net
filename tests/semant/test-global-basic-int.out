int x = (int : 45);
int y = (int : -6);
int z = (int : 33);
int a = (int : 0);
int b = (int : 1);
int c = (int : 0);
int d = (int : 1);
int main()
{
return (int : 0);
}

